test/stat_tracker_test.rb

```diff
@@ -22,6 +23,33 @@ def test_it_exists
     assert_instance_of StatTracker, @stat_tracker
   end
 
+  def test_winningest_coach_with_mock
+    locations = {
+      games: 'no path',
+      teams: 'no path',
+      game_teams: 'no path'
+    }
+
+    stat_tracker = StatTracker.from_csv(locations)
+
+    game_team_1 = mock(game_id: 2012001, head_coach: "Tom", result: "WIN")
```

Comment:
I think this could work as:

`game_team_1 = stub(game_id: 2012001, head_coach: "Tom", result: "WIN")`

I recommend this change because of this [article](https://www.codewithjason.com/rspec-mocks-stubs-plain-english/) and this [PR](https://github.com/hashmaster3k/futbol/pull/88)